tasks/main.yml

```diff
@@ -79,45 +79,64 @@
                 size: "{{ __hpc_usrlv_size }}"
                 mount_point: "{{ hpc_usrlv_mount }}"
 
-- name: Enable proprietary nvidia-driver
+- name: Update RHUI packages from Microsoft repositories only
```

Comment:
Hi @phagara, do you know why role needs to have only "rhui-microsoft-*" repos enabled in this step?
I guess we explicitly want the version that azure provides, not some other version from AppStream, even if AppStream version is newer?